sentry/templates/deployment-sentry-web.yaml

```diff
@@ -152,7 +170,9 @@ spec:
       {{- if .Values.sentry.web.customCA }}
       - name: custom-ca
         secret:
-{{ toYaml .Values.sentry.web.customCA | indent 10 }}
```

Comment:
Source of ValidationError(Deployment.spec.template.spec.volumes[2].secret): unknown field "item"